### ISSUE TITLE
Multiple languages support

### ISSUE THREAD

**Issue by jooray** [soupslurpr/Transcribro#27]
An ability to use multilingual models with directing language detection. 

Saying I want to recognize English and Slovak language. Whisper has good language detector, but for shorter texts it sometimes mistakes similar language (Slovak, Czech, Slovenian), so allowing to narrow down the choice would be great.

It would be amazing to be able to use this project for those of us who routinely communicate in more than one language.

**Comment by MXC48** [comment]
It's will be good to add more linguage support like french (for me) italian, etc

**Comment by soupslurpr** [comment]
Yes, Whisper supports those languages and many more so they will all be supported (as long as Whisper does good enough on them) once this feature is implemented.

**Comment by soupslurpr** [comment]
Moved to https://github.com/soupslurpr/Transcribro/issues/18

**Comment by Semuria** [comment]
Please add also german language. This app is amazing and it would be very helpfull.
